### ISSUE TITLE
plugin import error with airflow 1.10.x & python 3.6

### ISSUE THREAD

**Issue by qingyuan18** [teamclairvoyant/airflow-rest-api-plugin#47]
hi team
    we met the plugin installation error, which start the airflow webserver after installing:

Traceback (most recent call last):
--
  | File "/usr/local/lib/python3.6/site-packages/backports/configparser/__init__.py", line 1159, in _unify_values
  | sectiondict = self._sections[section]
  | KeyError: 'rest_api_plugin'

Am I missing something?

**Comment by rssanders3** [comment]
Hello @qingyuan18, can you confirm if you added the 'rest_api_plugin' section to the airflow.cfg file described in step #5 of the **Deployment Instructions** in the README (https://github.com/teamclairvoyant/airflow-rest-api-plugin/blob/master/README.md)?

**Comment by qingyuan18** [comment]
morning Sanders, yes I have added it into airflow cfg at the last line and leave other options as default
[rest_api_plugin]

we are running airflow on openshift , does that matter or it's really a airflow version compatibility issue?

**Comment by qingyuan18** [comment]
morning @rssanders3  Sanders, yes I have added it into airflow cfg at the last line and leave other options as default
[rest_api_plugin]

we are running airflow on openshift , does that matter or it's really a airflow version compatibility issue?

**Comment by qingyuan18** [comment]
hello @rssanders3  ,  is there any thing that I can do to continue my journey of airflow rest api plugin?

any reference would be better

**Comment by rssanders3** [comment]
@qingyuan18 can you confirm if there's anything more to the stack trace that you haven't included to help us identify at which point in the code it is failing? Or is that everything you get?

**Comment by qingyuan18** [comment]
hi @rssanders3  please find the followingl the logs related：

[2019-07-26 09:35:04,212] {__init__.py:51} INFO - Using executor CeleryExecutor
--
  | [2019-07-26 09:35:04,803] {plugins_manager.py:143} ERROR - No section: 'rest_api_plugin'
  | Traceback (most recent call last):
  | File "/usr/local/lib/python3.6/site-packages/backports/configparser/__init__.py", line 1159, in _unify_values
  | sectiondict = self._sections[section]
  | KeyError: 'rest_api_plugin'
  |  
  | During handling of the above exception, another exception occurred:
  |  
  | Traceback (most recent call last):
  | File "/usr/local/lib/python3.6/site-packages/airflow/plugins_manager.py", line 137, in <module>
  | m = imp.load_source(namespace, filepath)
  | File "/usr/local/lib/python3.6/imp.py", line 172, in load_source
  | module = _load(spec)
  | File "<frozen importlib._bootstrap>", line 684, in _load
  | File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  | File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  | File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  | File "/usr/local/airflow/plugins/rest_api_plugin.py", line 39, in <module>
  | log_loading = configuration.getboolean("rest_api_plugin", "LOG_LOADING") if configuration.has_option("rest_api_plugin", "LOG_LOADING") else False
  | File "/usr/local/lib/python3.6/site-packages/airflow/configuration.py", line 326, in has_option
  | self.get(section, option, fallback=_UNSET)
  | File "/usr/local/lib/python3.6/site-packages/airflow/configuration.py", line 283, in get
  | self.airflow_defaults.get(section, key, **kwargs))
  | File "/usr/local/lib/python3.6/site-packages/backports/configparser/__init__.py", line 792, in get
  | d = self._unify_values(section, vars)
  | File "/usr/local/lib/python3.6/site-packages/backports/configparser/__init__.py", line 1162, in _unify_values
  | raise NoSectionError(section)
  | backports.configparser.NoSectionError: No section: 'rest_api_plugin'
  | [2019-07-26 09:35:04,805] {plugins_manager.py:144} ERROR - Failed to import plugin /usr/local/airflow/plugins/rest_api_plugin.py
  | DB: postgresql+psycopg2://airflow:***@postgres:5432/airflow
  | [2019-07-26 09:35:05,133] {db.py:350} INFO - Creating tables

**Comment by tenapril** [comment]
im experiencing the same issue with airflow 1.10.3 and rest_api_plugin

```[2019-07-29 15:46:18,029] {plugins_manager.py:143} ERROR - No section: 'rest_api_plugin'
Traceback (most recent call last):
  File "/home/ubuntu/airflowve/lib/python3.6/site-packages/backports/configparser/__init__.py", line 1159, in _unify_values
    sectiondict = self._sections[section]
KeyError: 'rest_api_plugin'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/airflowve/lib/python3.6/site-packages/airflow/plugins_manager.py", line 137, in <module>
    m = imp.load_source(namespace, filepath)
  File "/home/ubuntu/airflowve/lib/python3.6/imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 684, in _load
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/ubuntu/airflow/plugins/rest_api_plugin.py", line 38, in <module>
    log_loading = configuration.getboolean("rest_api_plugin", "LOG_LOADING") if configuration.has_option("rest_api_plugin", "LOG_LOADING") else False
  File "/home/ubuntu/airflowve/lib/python3.6/site-packages/airflow/configuration.py", line 326, in has_option
    self.get(section, option, fallback=_UNSET)
  File "/home/ubuntu/airflowve/lib/python3.6/site-packages/airflow/configuration.py", line 283, in get
    self.airflow_defaults.get(section, key, **kwargs))
  File "/home/ubuntu/airflowve/lib/python3.6/site-packages/backports/configparser/__init__.py", line 792, in get
    d = self._unify_values(section, vars)
  File "/home/ubuntu/airflowve/lib/python3.6/site-packages/backports/configparser/__init__.py", line 1162, in _unify_values
    raise NoSectionError(section)
backports.configparser.NoSectionError: No section: 'rest_api_plugin'
[2019-07-29 15:46:18,030] {plugins_manager.py:144} ERROR - Failed to import plugin /home/ubuntu/airflow/plugins/rest_api_plugin.py
```

**Comment by etual118** [comment]
I experienced this issue when I use plugin file master, but it isn't show plugin file version v1.0.0.
My environment is python2.7 & airflow 1.10.3

**Comment by sudhakar-nallam** [comment]
This issue is resolved in #48
